themes/frontierline/content-views/content-mini.php

```diff
@@ -2,17 +2,23 @@
 /**
  * Display a post with just the thumbnail, title, and date.
  */
+$post_classes = 'post-mini';
+if (get_theme_mod('frontierline_post_thumbnail') !== '1') {
+  $post_classes .= ' post-thumb-disabled';
+}
 ?>
 
-<div class="post-mini">
+<div class="<?php echo $post_classes?>">
```

Comment:
Question: shouldn't be [`post_class`](https://codex.wordpress.org/Function_Reference/post_class) used here?

Reply:
I was just copying what was already here but `post_class` is definitely the best practice. I'll try it and see if anything breaks.